### DISCUSSION TITLE
Is there any settings to let WindTerm slience?

### DISCUSSION THREAD

**Discussion by superrrpear** [kingToolbox/WindTerm#655] [Q&A] [answered]
Hi, this is an really awosome tools! Thank you for your hard work!!!

I want to ask is there any settings to let WindTerm slience? I used it on Windows, and when I use <Tab> to automatic complete some linux command, I will get the "Diling Diling" sounds from Windows sometimes. As I know, some terminal can be set to let the terminal be quiet, even if sometimes we enter the wrong letter or some commands actually not exist when using <Tab>.

But I do not find any settings about sound in WindTerm, it there any way to mute WindTerm?

Thank you so much! Really appreciate!

**Comment by superrrpear** [comment, marked as answer]
Oh, I just saw there is "How to remove the beep sound in WindTerm #654" in the [Issues], and seem like it will release on 2.5.0 version? Thank you!

Just paste the workaround from author, you can use this as the workaround, or wait the 2.5.0 version.

(https://github.com/kingToolbox/WindTerm/issues/654#issuecomment-1098090023)

**Reply by kingToolbox** [reply]
The `2.4.0` version has completed most of the planned features and too many unplanned features, and is stable enough. If I don't stop, we'll never be able to start the new `2.5.0` version. Many users do not use the beta version for stability reasons, so stopping and releasing the official `2.4.0` version can also allow them to use new features.

In the new `2.5.0` version, I will bring better `Filer`, `Sender`, `QuickBar`, `SSH agent`, and maybe `AutoComplete`.
